src/contexts/bridge.context.tsx

```diff
@@ -139,7 +139,10 @@ const BridgeProvider: FC = (props) => {
         throw new Error("Connected provider is not available");
       }
 
-      const contract = Bridge__factory.connect(from.contractAddress, connectedProvider.getSigner());
+      const contract = Bridge__factory.connect(
+        from.contractAddress,
+        connectedProvider.provider.getSigner()
```

Comment:
Why we access `connectProvider.provider` instead of just `connectedProvider` as we were doing before?